qlever_dumps/props/props_text.py

```
@@ -0,0 +1,458 @@
+        # ---
+        if len(rows) < max_v:
+            # ---
+            # usage = prop_tab.get("items_use_it", 0)
+            usage = prop_tab.get("len_prop_claims", 0)
+            # ---
+            old_prop = Old_props.get(prop, {})
+            # ---
+            old_usage = old_prop.get("len_prop_claims")
+            diff = min_it(usage, old_usage, add_plus=True)
+            # ---
+            value_in_most_props = most_props.get(prop, 0)
```

Ruff (0.13.1)

Undefined name `most_props`

(F821)